app/views/communities/show.html.slim

```diff
@@ -1,9 +1,14 @@
 = stylesheet_link_tag 'communities'
-
-h1 class="d-flex justify-content-center"
-  = @community.title
+span
+  h1 class="d-flex justify-content-center"
+    = @community.title
+    span
+      - if policy(@community).update?
+        = link_to "Edit", edit_community_path(@community.id), class: "btn btn-primary btn-sm"
+      - if policy(@community).destroy?
+        = link_to "Delete", community_path(@community.id), method: :delete, class: "btn btn-danger btn-sm", data: { confirm: "Are you sure?" }
 hr
-- if user_signed_in?
+- if policy(:post).new?
```

Comment:
`- if policy(Post.new(user: current_user)).new?` man labāk patiktu neanonīmie veidi, kā autorizēt.

Reply:
Ok , es vienkarsi kad lasiju par gem , tur bija teikts par to ka user pec default esot  current_user , tapec tam baigi uzmanibu nenoveleju